site/src/App/Navigation/Navigation.tsx

```diff
@@ -105,7 +105,9 @@ const PreviewBranchPanel = () => {
             )}
           >
             <MenuItemLink
-              href={`https://seek-oss.github.io/braid-design-system${pathname}/${hash}`}
+              href={`https://seek-oss.github.io/braid-design-system${pathname}${
+                !pathname.endsWith('/') ? '/' : ''
+              }${hash}`}
```

Comment:
Yep, works as expected, will update.

```suggestion
              href={`https://seek-oss.github.io/braid-design-system${pathname}${hash}`}
```